ADR/nnnn-exclude-events-api-from-user-rbac.md

```diff
@@ -0,0 +1,33 @@
+# NN. Exclude Kubernetes Events API from User RBAC Roles
+
+Date: 2025-09-05
+
+## Status
+
+Proposed
+
+Relates to [ADR 11. Roles and Permissions](0011-roles-and-permissions.md)
+
+## Context
+
+Konflux users require RBAC permissions to interact with various Kubernetes resources through our defined roles (Viewer, Contributor, Maintainer, Admin). The Kubernetes `events` API provides access to cluster events, which can contain information about resource changes and system activities. Users may find that API useful, but it raises other problems.
+
+## Decision
+
+We will exclude the Kubernetes `events` API from all Konflux user RBAC roles.
```

Comment:
What does this mean for resourc-attached events like the ones we are using to e.g. report on ProjectDevelopmentStreamTemplate application status?

Reply:
project-controller only has events RN, so we can't really implement this unless we make an alternative.

Reply:
Ack.